statsmodels/stats/proportion.py

```diff
@@ -123,6 +123,183 @@ def func(qi):
         raise NotImplementedError('method "%s" is not available' % method)
     return ci_low, ci_upp
 
+def multinomial_proportions_confint(counts, alpha=0.05, method='goodman'):
+    '''Confidence intervals for multinomial proportions.
+
+    Parameters
+    ----------
+    counts : array_like of int
```

Comment:
Actually, I ran into some errors with the sison-glaz method when not using int (it can't find a value for `c`). I think it makes sense as the method is really only defined for ints.

Should we investigate? Or say `array_like of int, 1-D`?